dvls-get-secret-entry/README.md

```diff
@@ -0,0 +1,85 @@
+# Devolutions Server Get SecretEntry Action
+
+This GitHub Action allows you to authenticate and retrieve a secret entry from Devolutions Server.
+
+## Prerequisites
+
+Before using this action, you must first obtain a token using the `dvls-login` action:
+
+```yaml
+steps:
+  - name: Login to Devolutions Server
+    uses: devolutions/dvls-login@v1
+    with:
+      server_url: 'https://your-server.devolutions.app'
+      app_key: ${{ secrets.DVLS_APP_KEY }}
+      app_secret: ${{ secrets.DVLS_APP_SECRET }}
+      # The token will be stored in DVLS_TOKEN by default
+```
+
+## Inputs
+
+| Input | Description | Required | Default |
+|-------|-------------|----------|---------|
+| `server_url` | URL of the Devolutions Server | Yes | - |
+| `token` | Token for authentication | Yes | - |
+| `vault_name` | Name of the vault containing the secret entry | Yes | - |
+| `entry_name` | Name of the secret entry to retrieve | Yes | - |
+| `output_variable` | Name of the environment variable to store the retrieved secret | No | `DVLS_ENTRY_SECRET` |
+
+## Usage
+
+```yaml
+steps:
+  - name: Get Secret from Devolutions Server
+    uses: devolutions-community/dvls-get-secret-entry@main
```

Comment:
old action name ?

Reply:
Done. I've removed the node modules, compiled and tested they still work.